packages/react-mutation-mapper/src/component/column/Annotation.tsx

```diff
@@ -131,7 +132,9 @@ export function getAnnotationData(
                 _.find(
                     oncoKbCancerGenes.result,
                     (gene: CancerGene) =>
-                        gene.oncokbAnnotated &&
+                        (mutation.mutationType ===
+                            MAIN_MUTATION_TYPE_MAP['fusion'].label ||
```

Comment:
use this one instead? https://github.com/cBioPortal/cbioportal-frontend/blob/cd06ecf60e07cb8ce2af6c9d0beda5225dee20fc/packages/cbioportal-frontend-commons/src/lib/getCanonicalMutationType.ts#L16